### PR TITLE
net-proxy/torsocks: fix build with musl libc

### PR DESCRIPTION
**Pull request by doughdemon**
@blueness

### CODE REVIEW

File: net-proxy/torsocks/torsocks-2.1.0-r1.ebuild [hunk 1]
```diff
@@ -40,9 +44,6 @@ src_configure() {
 src_install() {
 	default
 
-	dodoc ChangeLog README.md TODO doc/notes/DEBUG doc/socks/{SOCKS5,socks-extensions.txt}
-
-	#Remove libtool .la files
-	cd "${D}"/usr/$(get_libdir)/torsocks || die
-	rm -f *.la
+	# Remove libtool .la files
+	rm -f "${ED}"/usr/$(get_libdir)/torsocks/*.la || die
```
Comment (SoapGentoo) [inline net-proxy/torsocks/torsocks-2.1.0-r1.ebuild]: Just use find, more idiomatic: https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives

**Comment by doughdemon** [comment]
Instead of `find "${D}" -name '*.la' -delete || die` shouldn't the
snippet say `find "${ED}" -name '*.la' -delete || die`? Also using
`-exec rm -rf {} +` would be more portable find usage. All four
combinations can be found in the tree. Could the wiki be corrected
to have the most preferred variant?

**Comment by SoapGentoo** [comment]
@doughdemon for find `$D` vs `$ED` makes no difference, as in the `$D` case it will just walk the `EPREFIX` tree regardless. Concerning `-delete` vs `-exec rm -rf {} +`, the former should be preferred, because 1) `-delete` is nicer and clearer to read 2) `.la` files will never be directories, hence won't need `-r` and 3) `EAPI=5` and above guarantee find to be GNU find, hence portability is not an issue here. So the wiki does have the most preferred, succinct version I believe.

**Comment by doughdemon** [comment]
thanks for the rationale! The pull request is updated.

**Comment by SoapGentoo** [comment]
@doughdemon also please rebase to latest master, in order to get rid of the QA message.

**Comment by blueness** [comment]
thanks pushed.

**Pull request closed without merging**